### ISSUE TITLE
Getting math on the poster

### ISSUE THREAD

**Issue by jpfairbanks** [mehalter/siam-pp20-poster#1]
One of the key points to make about the system is that wiringdiagrams, because they allow abstraction allow you to get more math per visual element than computer algebra systems alone. I’d like a way to illustrate that there are a lot of equations and algebraic terms associated with the malaria model as a DiffEQ, even though the algebraic representation as a morphism is very compact.

**Comment by mehalter** [comment]
I added the initial diffeqs of the lotka-volterra model and the malaria model to the poster and said that the wiring diagram condenses that down. I don't think we should put the math from the combined model because its too much text, but when we are talking about the poster we can mention that and it helps to show the relation between the structured layer and the number of differential equations, and then when you see the complex structured layer for the final model, the magnitude of the number of differential equations can be easily imagined.

**Comment by jpfairbanks** [comment]
Yeah, maybe we should just report the number of terms in the final eqns because you are right that it will be a waste of space to show them.
